### ISSUE TITLE
OnlyOffice no Linux (OnlyOffice in Linux Version)

### ISSUE THREAD

**Issue by luanrafael19** [ONLYOFFICE/DesktopEditors#8]
O OnlyOffice na versão linux não possui dicionário na língua portuguesa. As palavras não podem ser acentuadas, não é possível utilizar estes caracteres: á, é, í, ó, ú, â, ê, î, ô, û, à, è, ì, ò, ù, ã, ẽ, ĩ, õ, ũ.
Se estes problemas forem concertados, o OnlyOffice tem tudo para ser o melhor app para Linux, inclusive melhor que o LibreOffice e WPS Office.

OnlyOffice the Linux version does not have dictionary in Portuguese. The words are unaccented, you can not use these characters: á, é, í, ó, ú, â, ê, î, ô, û, à, è, ì, ò, ù, ã, í, õ, ũ.
If these problems are concerted, the OnlyOffice has everything to be the best app for Linux, even better than LibreOffice and WPS Office.

Sorry my bad english!

**Comment by maxkadushkin** [comment]
there are 2 dictionaries for Portuguese in app - Brazilian and Portugal. are they unsuitable?

**Comment by luanrafael19** [comment]
Sorry, they are correct. But I can't write this on Only Office: á, é, í, ó, ú, â, ê, î, ô, û, à, è, ì, ò, ù, ã, ẽ, ĩ, õ, ũ. I think OnlyOffice does not support the ABNT2 keyboard.
Thanks for answering me!!
Sorry for my bad english

**Comment by maxkadushkin** [comment]
yes, we detected this bug. unfortunately, bug in 3dparty library, it seems in CEF framework. and we have no opportunity to fix it yet, looking forward updates for CEF for awhile

**Comment by maxkadushkin** [comment]
we posted [bug](https://bitbucket.org/chromiumembedded/cef/issues/2039) on CEF

**Comment by ViktorOn** [comment]
Is this bug already fixed? 4 years have already passed, why is the request still open?

**Comment by ShockwaveNN** [comment]
@ViktorOn As we said in 2016 it's a problem in CEF

And it CEF [this bug](https://bitbucket.org/chromiumembedded/cef/issues/2039), which is duplicate of this one(https://bitbucket.org/chromiumembedded/cef/issues/1813/cef-does-not-work-with-imes-on-linux) - is still actual, last activity in November of 2021 and `New` status

Nothing we can do here

**Comment by ViktorOn** [comment]
> @ViktorOn As we said in 2016 it's a problem in CEF
> 
> And it CEF [this bug](https://bitbucket.org/chromiumembedded/cef/issues/2039), which is duplicate of [this one](https://bitbucket.org/chromiumembedded/cef/issues/1813/cef-does-not-work-with-imes-on-linux) - is still actual, last activity in November of 2021 and `New` status
> 
> Nothing we can do here

I understand, the question is removed.
